README.md

```diff
@@ -17,8 +17,8 @@ Credentials within this project are managed using environment variables and [Git
 # Access key from Stripe to access your Stripe resources
 STRIPE_KEY=STRIPE_KEY
 
-# The product ID which users are subscribed to.
-EMAIL_PRODUCT_ID=STRIPE_KEY
+# The ID of the price product object used in billing subscribers.
+EMAIL_PRODUCT_PRICE_ID=STRIPE_KEY
```

Comment:
The name of this variable has been renamed to `EMAIL_PRODUCT_PRICE_ID` since the merge of this [pull request](https://github.com/ambianic/ambianic-subscriptions/pull/19) to better indicate its usage.

The `EMAIL_PRODUCT_PRICE_ID` here has no relation to a user's account. It is used within the Stripe implementation details.

A **subscription** entity created for a user is tied to a **price** entity which is tied to a **product** entity. i.e `SUBSCRIPTION_ID -> PRICE_ID -> PRODUCT_ID`.

See docs: [Create Subscription Entity](https://stripe.com/docs/api/subscriptions/create), [Create Price Entity](https://stripe.com/docs/api/prices/create)